pkgs/by-name/ve/vesktop/package.nix

```diff
@@ -77,7 +77,7 @@ stdenv.mkDerivation (finalAttrs: {
       dontBuild = true;
       dontFixup = true;
       outputHashMode = "recursive";
-      outputHash = "sha256-cnk+KFdvsgG1wGDib7zgIS6/RkrR5EYAHtHcrFSU0Es=";
+      outputHash = "sha256-y7lOkFNcsJlRCiTdTb5AW4a5WXov2+bYyHjuXJHAubQ=";
```

Comment:
```suggestion
      outputHash = "sha256-JLjJZYFMH4YoIFuyXbGUp6lIy+VlYZtmwk2+oUwtTxQ=";
```

Reply:
thanks!!! still figuring out how hashes work in Nix. this is my first time contributing here 😅